open_schools_platform/user_management/users/serializers.py

```diff
@@ -0,0 +1,66 @@
+from re import match
+
+from phonenumber_field.serializerfields import PhoneNumberField
+from rest_framework import serializers
+
+
+class CreationTokenSerializer(serializers.Serializer):
+    phone = PhoneNumberField(
+        max_length=17,
+        required=True,
+    )
+    recaptcha = serializers.CharField(
```

Comment:
Should we add serializer on recaptcha? 
Actually, firebase can do it instead of us